src/views/MainView.ts

```diff
@@ -0,0 +1,40 @@
+import { $ } from "../utils/domSelectors";
+import renderList from "../components/RestaurantList";
+import { Restaurant } from "../types/types";
+
+class MainView {
+  private addButton = $(".gnb__button") as HTMLButtonElement;
+  private modal = $(".modal") as HTMLDialogElement;
+  private categoryFilter = $("#category-filter") as HTMLSelectElement;
+  private sortingFilter = $("#sorting-filter") as HTMLSelectElement;
```

Comment:
요 부분 이 **HTMLElement에 접근할 때마다 `as`로 타입 단언** 에 대해 질문주신 부분 이 맞을까요? as를 사용할 수도 있는 부분이고, generic을 이용해서 다음과 같이 표현할 수 있지 않을까 싶어요. 다만 `$` 라는 유틸을 사용하고 계시기 때문에 해당 함수가 제네릭을 받을 수 있도록 설정해 주어야 합니다. 요 부분은 추가로 학습해보시면 좋을 것 같아요~

```suggestion
  private addButton = $<HTMLButtonElement>(".gnb__button");
```